src/components/option-select/option-select.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+import './option-select.css';
+
+const OptionSelect = ({ options = [], selection, onSelect }) => {
```

Comment:
As I understand the task, you should take `selection` and `onSelect` as props, and use them, instead of local state. Otherwise both of them, currently, seem redundant here. But, your implementation makes more sense since you'd want to hold this data locally, so I'll accept it